applications/ALEapplication/python_scripts/mesh_solver_base.py

```diff
@@ -127,4 +127,4 @@ def _create_mesh_motion_solver(self):
         
         The mesh motion solver must provide the functions defined in SolutionStrategy.
         """
-        raise Exception("Mesh motion solver must be created by the derived class.")
+        raise Exception("Mesh motion solver must be created by the derived class.")
```

Comment:
@elaf if you could add the newlines you removed in this and the next file then its ok from my side

Reply:
Greetings dear @philbucher and very good day to you,

kindly find updated changes, I used exactly the same files from the master branch,

thank you for your usual support we highly appreciate it, and kindly accept my apology for any inconvenience; honestly, this is my first experience with git and pull request,

best regards,